lib/ansible/modules/packaging/os/yum.py

```diff
@@ -1095,7 +1125,7 @@ def ensure(module, state, pkgs, conf_file, enablerepo, disablerepo,
                     for i in new_repos:
                         if not i in current_repos:
                             rid = my.repos.getRepo(i)
-                            a = rid.repoXML.repoid
+                            a = rid.repoXML.repoid  ##  if no one complains remove this on next MR 
```

Comment:
Check the yum code for sideeffects (repoid might be a property and referencing it might cause some action to happen).  If this is a simply attribute rather than something producing side-effects, feel free to get rid of it.